src/Api/Controller/Changes.php

```diff
@@ -77,6 +91,8 @@ public static function save(ModelWithContent $model, array $input): array
 		$changes = $model->version(VersionId::changes());
 		$latest  = $model->version(VersionId::latest());
 
+		static::cleanup($model);
```

Comment:
`Todo:` Can we add comments here to clarify as I was writing a long question only to realize at the end why this works? Something like "it's save to remove the legacy lock file at any call to this method as all invalid calls, e.g. from another users, will be blocked through the existing legacy lock. The only actions that can call this method can origin from the lock owner which then should clean up the legacy lock". Maybe better formulated, but to make this easier to understand later on.

Plus, we should add `@todo` marks to remove this in v6, v7 or so.